docs/docs/modeling.md

```diff
@@ -106,6 +106,41 @@ classDiagram
 
 ## ShareHouse - シェアハウス
 
+The Landlord can register the ShareHouse owned by the Landlord.
+
+### ShareHouse ID (Identifier)
+
+The ShareHouse ID is the UUID.
+
+### ShareHouse Name
+
+The name of the ShareHouse can be set and changed.
+The ShareHouse has a default name when created without specifying one.
+
+Constraints：The ShareHouse Name must be
+
+- at least 1 character and no more than 15 characters long.
```

Comment:
Could you add this constraint?
"The ShareHouse name must be unique."
Reference: 
https://www.notion.so/Functional-Requirements-409ee1aa29074c65a88b78c8beee44cb#a5aed299afa54ba99119712d4db2dc26